lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1290,10 +1290,16 @@ static bool performCompile(CompilerInstance &Instance,
   if (opts.PrintClangStats && Context.getClangModuleLoader())
     Context.getClangModuleLoader()->printStatistics();
 
-  emitReferenceDependenciesForAllPrimaryInputsIfNeeded(Instance);
   emitSwiftRangesForAllPrimaryInputsIfNeeded(Instance);
   emitCompiledSourceForAllPrimaryInputsIfNeeded(Instance);
```

Comment:
@davidungar Am I correct in assuming that emitSwiftRangesForAllPrimaryInputsIfNeeded() and emitCompiledSourceForAllPrimaryInputsIfNeeded() don't need to wait until after SILGen/IRGen?

Indeed emitCompiledSourceForAllPrimaryInputsIfNeeded() doesn't appear to be used at all.